frontend/app/src/components/inputs/checkbox.tsx

```diff
@@ -21,7 +21,7 @@ export const Checkbox = forwardRef<HTMLInputElement, CheckboxProps>((props, ref)
       className={classNames(
         "focus:ring-0 focus:ring-offset-0",
         focusStyle,
-        "w-4 h-4 text-custom-blue-800 disabled:text-gray-300 bg-gray-100 border-gray-300 rounded cursor-pointer disabled:cursor-not-allowed"
+        "w-4 h-4 mr-2 text-custom-blue-800 disabled:text-gray-300 bg-gray-100 border-gray-300 rounded cursor-pointer disabled:cursor-not-allowed"
```

Comment:
adding margin inside a primitive component is not recommended. If you need some space, the preferred way is to style parent or by using className prop.